api/client/service/opts.go

```diff
@@ -235,7 +235,8 @@ func (m *MountOpt) Type() string {
 func (m *MountOpt) String() string {
 	mounts := []string{}
 	for _, mount := range m.values {
-		mounts = append(mounts, fmt.Sprintf("%v", mount))
+		repr := fmt.Sprintf("%s %s %s", mount.Type, mount.Source, mount.Target)
```

Comment:
The `String()` for a `pflag.Value` type is used to represent the stored value (ex: when the default value is displayed). It's not necessarily designed to echo out the format of the flag that is parsed by `Set()`.

This `String()` is unlikely to ever be used, because I don't think we'll ever have a default value for a mount flag, but I thought that this would provide a decent minimum amount of information for the rare case that someone needs to print one of these for debugging.

Would you prefer going back to the `%v`?